### PR TITLE
making flocking and gliding more open

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

File: uw-research-computing/scaling-htc.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ Contents
 ================================================
 
 Running on other resources in addition to CHTC has one huge benefit:
-size! The UW Grid and OS Pool include thousands of computers,
-addition to what\'s already available in CHTC. Most CHTC users who run
+size! The UW Grid and Open Science Pool (OS Pool) include thousands of computers,
+addition to what\'s already available in CHTC, including specialized
```
Comment (ChristinaLK) [inline uw-research-computing/scaling-htc.md:29]:
```suggestion
in addition to what\'s already available in CHTC, including specialized 
```

File: uw-research-computing/scaling-htc.md [hunk 2]
```diff
@@ -73,18 +73,21 @@ Not all jobs will run well outside of CHTC. Because these jobs are
...
     transfer guide](file-availability) and our [SQUID
     guide](file-avail-squid)), and output files are small enough
-    to return to your home directory (less than 2 GB). We normally
-    handle larger files in our large data filesystem (see our [data
-    staging guide](file-avail-largedata)) which is only available
-    from CHTC.  
+    to return to your home directory (less than 2 GB).  If your data files 
+    are large enough to use our large data file system ("staging", see our [data
+    staging guide](file-avail-largedata)) and you would like to use external
+    resource, please [contact us](chtc@cs.wisc.edu)!  In certain cases, we may be
```
Comment (ChristinaLK) [inline uw-research-computing/scaling-htc.md:83]:
```suggestion
    resources, please [contact us](chtc@cs.wisc.edu)!  In certain cases, we may be 
```

File: uw-research-computing/scaling-htc.md [hunk 3]
```diff
@@ -48,20 +48,14 @@ We call sending jobs to other pools on campus *flocking*.
...
-any unused computers that are available to run jobs. These unused
-computers then form a pool of resources where jobs can go run.
+CHTC maintains an OSG pool for the campus community, which includes 
+resources contributed by campuses, national labs, and other institutions 
+across and beyond the US.
 
-CHTC is a member of the OSG Consortium, so our submit servers,
-besides sending jobs to CHTC computers (the default), can send jobs to
-the OSG. We call sending jobs to other institutions *gliding*.
+We call sending jobs to other institutions in our OSG pool *gliding*.
```
Comment (lmichael107) [inline uw-research-computing/scaling-htc.md:58]:
```suggestion
When you send jobs to other institutions in our OSG pool, we call that *gliding*.
```

File: uw-research-computing/scaling-htc.md [hunk 4]
```diff
@@ -48,20 +48,14 @@ We call sending jobs to other pools on campus *flocking*.
 
 <a name="osg"></a>
 
-B. The OSG Consortium
+B. OSG Pool
```
Comment (lmichael107) [inline uw-research-computing/scaling-htc.md:51]:
```suggestion
B. UW-Madison's OSG Pool
```

**Review by lmichael107 — CHANGES_REQUESTED** [review]

File: uw-research-computing/scaling-htc.md [hunk 5]
```diff
@@ -54,7 +54,7 @@ B. The OSG Consortium
 The <a href="http://www.osg-htc.org/">OSG Consortium</a> is a group of universities and research labs
 who have agreed to share their unused computational resources with each
 other. If a job is submitted from an OSG submission point, it can run in
-the [OS pool](https://osg-htc.org/about/open_science_pool/) associated with that submission point. This OSG pool is the
+the [Open Science Pool](https://osg-htc.org/about/open_science_pool/) associated with that submission point. This OSG pool is the
```
[the remark this answers is not in the document]
Comment (ChristinaLK) [reply uw-research-computing/scaling-htc.md]: Nope, this is from when Cannon purged Open Science Grid from the website four months ago.

File: uw-research-computing/scaling-htc.md [hunk 6]
```diff
@@ -73,18 +73,21 @@ Not all jobs will run well outside of CHTC. Because these jobs are
 running all over the campus or country, on computers that don\'t belong
 to us, they have two major requirements:
 
--   **No \"large\" data**: All of the files for your jobs are in your
-    home directory and/or in SQUID (the options listed on our [file
+-   **Moderate Data Sizes**: Ideally, all of the files for your jobs should be
+    in your home directory and/or in SQUID (the options listed on our [file
```
[the remark this answers is not in the document]
Comment (ChristinaLK) [reply uw-research-computing/scaling-htc.md]: yes, and my latest changes should have that information.

**Comment by ChristinaLK** [comment]
@lmichael107 it looks like you might have seen an older version of this PR when reviewing. I incorporated most of your suggestions - can you give it one more once-over?

**Pull request merged**